datalad/downloaders/http.py

```diff
@@ -65,6 +65,17 @@
 __docformat__ = 'restructuredtext'
 
 
+def process_www_authenticate(v):
```

Comment:
Could you please add a short docstring to describe the purpose of the function and why output should be a list? (that is should provide value for the `supported_types`)

Reply:
Uhh you wrote that code.  I don't know its purpose or why its output should be a list, but, sure, I could guess and add a docstring.  In the time it took you to "review" the function you wrote, you could have wrote that docstring.

Reply:
ah, damn me -- should've added a docstring indeed, will do